Essentials/src/com/earth2me/essentials/ReserveEssentials.java

```diff
@@ -0,0 +1,1830 @@
+package com.earth2me.essentials;
+
+import com.earth2me.essentials.api.NoLoanPermittedException;
+import com.earth2me.essentials.api.UserDoesNotExistException;
+import net.ess3.api.Economy;
+import net.tnemc.core.Reserve;
+import net.tnemc.core.economy.EconomyAPI;
+import net.tnemc.core.economy.currency.Currency;
+import org.bukkit.Bukkit;
+import org.bukkit.World;
+
+import java.math.BigDecimal;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+
+import static com.earth2me.essentials.I18n.tl;
+
+/**
+ * @author creatorfromhell <daniel.viddy@gmail.com>
+ */
+public class ReserveEssentials implements EconomyAPI {
+
+    private Essentials plugin;
```

Comment:
```suggestion
    private final Essentials ess;
```

Consistency with the rest of EssentialsX, and there shouldn't be any reason for this to change.